### DISCUSSION TITLE
App alerting for applications with no permission

### DISCUSSION THREAD

**Discussion by bobbythomas** [kamaravichow/safe-dot-android#22] [Q&A] [answered]
Hi, Aravind

I am not sure why this app is alerting when I open all apps. This is alerting even if an app isn't granted any permission. Is this how it works? Is there a way to view the logs shown in the app?

Thank you,
Regards,
Bob

**Comment by kamaravichow** [comment, marked as answer]
When any permission is in use, my app checks for the app which is currently running in the foreground. 

In your case, I think some other app is trying to access your sensors when another app is running in foreground(which doesn't have any permissions) which makes my app think that the app in the foreground is using those sensors.

So don't completely depend on the access logs, as for now we cannot get the information about what app is exactly using your sensors.

Feel free to ask if you didn't understand . Thanks for using my app.

**Reply by kehtabp** [reply]
Thanks for the answer, but it's misleading and should be explained in the app!
